resources/js/components/FormField.vue

```diff
@@ -9,6 +9,7 @@
       <draggable
         v-model="items"
         handle=".relationship-item-handle"
+        :disabled="!field.sortable"
```

Comment:
Add a space after "not"
```suggestion
        :disabled="! field.sortable"
```